cli/crates/backend/src/api/publish.rs

```diff
@@ -35,7 +35,22 @@ pub async fn publish(
         },
     });
 
-    let cynic::GraphQlResponse { data, errors } = client.post(api_url()).run_graphql(operation).await?;
+    let resp = client
+        .post(api_url())
+        .json(&operation)
+        .send()
+        .await
+        .unwrap()
```

Comment:
Did you mean to leave in these `unwrap`s and `println`?

Reply:
wow hum no idea why it's in this PR, rebase didn't work as expected. good catch, thx